cves/CVE-2013-2277.yml

```diff
@@ -169,10 +183,13 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked. Thus, 'answer' should always have some
     explanation.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
```

Comment:
See my comment on the same section in the previous CVE.

Reply:
I will change those to false.